src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -166,9 +168,11 @@ public static void init()
         makeRegistry(WORLD_CARVERS, c(WorldCarver.class)).disableSaving().disableSync().create();
         makeRegistry(SURFACE_BUILDERS, c(SurfaceBuilder.class)).disableSaving().disableSync().create();
         makeRegistry(FEATURES, c(Feature.class)).addCallback(FeatureCallbacks.INSTANCE).disableSaving().create();
+        makeRegistry(CONFIGURED_FEATURES, c(ConfiguredFeature.class)).addCallback(FeatureCallbacks.INSTANCE).disableSaving().create();
```

Comment:
I believe both the configured features and configured structure features are also dynamic registries so this potentially need to be done in a more similar way to biomes being done, unless the reason biomes are like they are forge registry wise because vanilla only has a dynamic registry for them and not both types of registries.

Reply:
So I would have to try to integrate this like this [commit ](https://github.com/MinecraftForge/MinecraftForge/commit/d8eaeabf3cf03b2de05c23ce88c3b75548720014)

Reply:
I think so

Reply:
I changed some thing, not the GameData but by testing with a DataPack adding structures and features, it works nicely